src/main/java/org/takes/facets/auth/codecs/CcAES.java

```diff
@@ -145,15 +118,40 @@ public Identity decode(final byte[] bytes) throws IOException {
     }
 
     /**
+     * Create AlgorithmParameterSpec with the block size.
+     * @return The AlgorithmParameterSpec
+     */
+    private static AlgorithmParameterSpec initAlgorithmParameterSpec() {
+        final SecureRandom random = new SecureRandom();
+        final byte[] bytes = new byte[CcAES.BLOCK];
+        random.nextBytes(bytes);
+        return new IvParameterSpec(bytes);
+    }
+
+    /**
+     * Check the block size of the key.
+     */
+    private void checkBlockSize() {
+        if (this.key.length != CcAES.BLOCK) {
+            throw new IllegalArgumentException(
+                String.format(
+                    "the length of the AES key must be exactly %d bytes",
+                    CcAES.BLOCK
+                )
+            );
+        }
+    }
+
+    /**
      * Decrypt the given bytes using AES.
-     *
```

Comment:
@sebing please bring this line back, see https://google.github.io/styleguide/javaguide.html#s7.1.2-javadoc-paragraphs